src/modules/reporting/components/reporting-header/reporting-header.jsx

```diff
@@ -28,8 +31,10 @@ export default class ReportingHeader extends Component {
       <article className={Styles.ReportingHeader}>
         <div className={Styles.ReportingHeader__header}>
           <div>
-            <h1 className={Styles.ReportingHeader__heading}>Reporting</h1>
-            <span className={Styles.ReportingHeader__endDate}>Reporting cycle ends { s.endDate }</span>
+            <h1 className={Styles.ReportingHeader__heading}>{p.heading}</h1>
+            { p.showReportingEndDate &&
```

Comment:
Should we also include the cycle graph as conditionally displayed?

Feels odd removing the date, but leaving the cycle graph.  I might be missing something however.

Reply:
On second thought — I’m going to leave this here and we’ll raise it during our discussion w/ Tara.

Per the mocks it’s there (which is where I lean), but I’d be curious to have a discussion before we make a change here.